_compdemos/gizmo-node-beta.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Gizmo Node Beta
+main_author: Michael Gutteridge
+primary_reviewers: atombaby
+---
+
+This page describes the new gizmo cluster nodes, some of the differences
+between these new systems, and how you can help bring these nodes into service.
+
+# The Hardware
+
+These new systems will be the "K" class of nodes.  Currently we have four in
```

Comment:
Should be more clear that we have four types of nodes, not four k nodes. Maybe link to the hardware table? Has that been updated with the k-node info?

Reply:
9d2759c